### ISSUE TITLE
Weird characters are being inserted into CLI

### ISSUE THREAD

**Issue by FrenchBen** [docker/dockercloud-cli#9]
### Issue
Some odd characters to the CLI on initial type

### Expected
A clean CLI which only shows what I type

### Actual
`1;7R` gets inserted in front of all keystroke

### Steps to reproduce
1. Open up a CLI into a container using `golang:1.5-alpine` e.g. : `docker-cloud exec alpine-test-1`
2. type `/go # hello world`
3. see that the actual output is `/go # 1;7Rhello world`

**Comment by tifayuki** [comment]
Yes, that happens on all the containers using alpine as the base image.
I think it is the control character from alpine.

@fermayo Anything we can do to solve this?
